mappings/servers/echomc/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "echomc",
+    "name": "EchoMC",
+    "description": "The best kitpvp server in italy",

```

Comment:
Names should start with a capital letter, so instead of `italy`, type `Italy` + it’s supposed to be `KitPvP`, not `kitpvp`